pkg/kubernetes/client.go

```diff
@@ -31,10 +31,10 @@ func init() {
 	} else {
 		kubeconfig = flag.String("kubeconfig", "", "absolute path to the kubeconfig file")
 	}
-	flag.Parse()
 }
 
 func getConfig() (*rest.Config, error) {
+	flag.Parse()
```

Comment:
This might mess up both `dapr mtls expiry` and `dapr mtls export`. Please verify those two commands work after this change.

Reply:
Tested working ... since it also calling `GetKubeConfigClient()` which calls `getConfig()` which calls for `flag.Parse()`

Reply:
$./dist/darwin_amd64/release/dapr mtls expiry
Root certificate expires in 8736 hours. Expiry date: 2021-08-17 22:13:29 +0000 UTC
$ ./dist/darwin_amd64/release/dapr mtls export
✅  Trust certs successfully exported to <pwd>

Reply:
For safety, we should use DoOnce to make sure we run it only once:

```go
doOnce.Do(func() {
flag.Parse()
})
```

Reply:
done